bin/start_fe.sh

```diff
@@ -199,7 +199,9 @@ if [ ${RUN_LOG_CONSOLE} -eq 1 ] ; then
         cp $STARROCKS_HOME/conf/fe.conf.readonly $STARROCKS_HOME/conf/fe.conf
     fi
     # force sys_log_to_console = true
-    echo -e "\nsys_log_to_console = true" >> $STARROCKS_HOME/conf/fe.conf
+    if ! grep sys_log_to_console $STARROCKS_HOME/conf/fe.conf | tr -d ' ' | grep -v "#sys_log_to_console"  &>/dev/null; then
```

Comment:
actually I am thinking about using environment variable instead of fe.conf configvar to control this behavior, so we don't have to make changes to fe.conf at all.

Reply:
this is also a small step forward to allow running starrocks in a readonly filesystem container environment.

Reply:
fixed